source/api/ut.pkb

```diff
@@ -64,6 +64,21 @@ create or replace package body ut is
   begin
     return ut_expectation_dsinterval(ut_data_value_dsinterval(a_actual), a_message);
   end;
+  
+  procedure fail(a_message in varchar2) is
```

Comment:
sometimes it familiar for user to set a point in code, if it's riched than something is wrong. The most frequent case (but not the only) is when exception is expceted.

Other frameworks also have such methods (http://junit.sourceforge.net/javadoc/org/junit/Assert.html)